rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -401,6 +401,68 @@ def __len__(self):
     def __del__(self):
         b.rocalRelease(self.loader._handle)
 
+
+class ROCALNumpyIterator(object):
```

Comment:
Same as generic.py file comment
Why don't we use the existing generic iterator with if condition for Numpy loader as required

Reply:
In numpy reader, there is no labels so we return only images. In generic iterator, even though label reader is not present, it returns label tensor which is set as None